modules/apps/message-boards/message-boards-service/src/test/java/com/liferay/message/boards/service/impl/MBMessageLocalServiceImplTest.java

```diff
@@ -0,0 +1,50 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2024 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.message.boards.service.impl;
+
+import com.liferay.message.boards.model.MBMessage;
+import com.liferay.message.boards.model.impl.MBMessageImpl;
+import com.liferay.portal.kernel.exception.PortalException;
+import com.liferay.portal.kernel.service.ServiceContext;
+import com.liferay.portal.kernel.test.ReflectionTestUtil;
+import com.liferay.portal.test.rule.LiferayUnitTestRule;
+
+import org.junit.Assert;
+import org.junit.ClassRule;
+import org.junit.Rule;
+import org.junit.Test;
+
+/**
+ * @author Mikel Lorza
+ */
+public class MBMessageLocalServiceImplTest {
+
+	@ClassRule
+	@Rule
+	public static final LiferayUnitTestRule liferayUnitTestRule =
+		LiferayUnitTestRule.INSTANCE;
+
+	@Test
+	public void testGetMessageURL() throws PortalException {
+		MBMessage mbMessage = new MBMessageImpl();
+
+		mbMessage.setMessageId(123456L);
```

Comment:
there is a reason to put this (123456L) instead RandomTestUtil.nextLong()?

Reply:
Absolutely not, changing it.

Reply:
I tried to use it but behind it we are using CounterLocalServiceUtil.increment(); so, we can not use it here since it is a j unit test, but we can use randomLong() instead of it.

Reply:
perfect! using autogenerated methods are better than fixed ones if there is no reason for a fixed value :)